chain/x/oracle/client/rest/query.go

```diff
@@ -128,6 +135,41 @@ func getRequestSearchHandler(cliCtx context.CLIContext, route string) http.Handl
 	}
 }
 
+func getRequestsPricesHandler(cliCtx context.CLIContext, route string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		decoder := json.NewDecoder(r.Body)
+		var requestPrices RequestPrices
+		err := decoder.Decode(&requestPrices)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+			return
+		}
+		prices := make([]pricecache.Price, len(requestPrices.Symbols))
+		height := int64(0)
+		for idx, symbol := range requestPrices.Symbols {
```

Comment:
If we request 25 or 50 symbols in one request how long it takes?

Reply:
50 symbols = 40ms